src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -70,7 +70,9 @@ internal sealed partial class ReflectTypeDescriptionProvider : TypeDescriptionPr
         // not merge them into the attribute set for a class.
         private static readonly Type[] s_skipInterfaceAttributeList = InitializeSkipInterfaceAttributeList();
 
-        [UnconditionalSuppressMessage ("ReflectionAnalysis", "IL2045:AttributeRemoval",
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2121:RedundantSuppression",
+            Justification = "The suppression is necessary when the feature is turned on")]
```

Comment:
Please add which feature switch this refers to (maybe mention that the attribute removal is conditional on that feature switch and thus when building with the feature switch enabled the removal is not present and thus the suppression is not necessary, but when it the feature switch is disabled, the suppression is necessary).